### PR TITLE
Add accessibility to confirmation view

### PR DESCRIPTION
**Pull request by gersonnoboa**
There's explanations inside the two commits.

MOB-1890

### CODE REVIEW

File: GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+CheckMessagesButtonStyle.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import Foundation
+import UIKit
```
Comment (yurii-glia) [inline GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+CheckMessagesButtonStyle.swift:2]: UIKit includes Foundation, so I'd like to propose to drop `Foundation`.
```suggestion
import UIKit
```

File: GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+SubtitleStyle.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import Foundation
+import UIKit
```
Comment (yurii-glia) [inline GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+SubtitleStyle.swift:2]:
```suggestion
import UIKit
```

**Review by yurii-glia — APPROVED** [review]

**Review by igorkravchenko — APPROVED** [review]

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+SubtitleStyle.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        public var font: UIFont
+        /// Title color.
+        public var color: UIColor
+        /// Accessibility related properties.
+        public var accessibility: Accessibility
+
+        /// - Parameters:
+        ///   - text: Title text value.
+        ///   - font: Title font.
+        ///   - color: Title color.
```
[the remark this answers is not in the document]
Comment (gersonnoboa) [reply GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+SubtitleStyle.swift:18]: Fixed 8f6f754a111d7bd7e6fcddcdaa4243e0db8c013b

**Comment by gersonnoboa** [comment]
!squash

File: GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+CheckMessagesButtonStyle.swift [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        /// Background color of the button.
+        public var backgroundColor: UIColor
+        /// Accessibility related properties.
+        public var accessibility: Accessibility
+
+        /// - Parameters:
+        ///   - title: Title text of the button.
+        ///   - font: Font of the title.
+        ///   - textColor: Color of the button title text.
+        ///   - backgroundColor: Background color of the button.
```
[the remark this answers is not in the document]
Comment (gersonnoboa) [reply GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+CheckMessagesButtonStyle.swift:21]: Fixed https://github.com/salemove/ios-sdk-widgets/commit/8f6f754a111d7bd7e6fcddcdaa4243e0db8c013b

**Review by rasmustautsglia — APPROVED** [review]

**Comment by gersonnoboa** [comment]
!merge

**Pull request merged**